src/compiler/types/tests/ComponentCompilerMeta.stub.ts

```diff
@@ -9,17 +9,85 @@ import * as d from '@stencil/core/declarations';
  */
 export const stubComponentCompilerMeta = (
   overrides: Partial<d.ComponentCompilerMeta> = {}
-): d.ComponentCompilerMeta => {
-  // TODO(STENCIL-378): Continue to build out default stub, remove the type assertion on `default`
-  const defaults: d.ComponentCompilerMeta = {
-    isCollectionDependency: false,
-    events: [],
-    methods: [],
-    properties: [],
-    sourceFilePath: '/some/stubbed/path/my-component.tsx',
-    tagName: 'stub-cmp',
-    virtualProperties: [],
-  } as d.ComponentCompilerMeta;
-
-  return { ...defaults, ...overrides };
-};
+): d.ComponentCompilerMeta => ({
+  assetsDirs: [],
+  componentClassName: '',
```

Comment:
Since we have a `tagName` of ['stub-cmp'](https://github.com/ionic-team/stencil/pull/3367/files#diff-e18c71115908470b2236d446a239b6524130d3a443cb5c672e66287fd1719f6aR89) it might make sense to give this a name like 'StubCmp'
```suggestion
  componentClassName: 'StubCmp',
```

FWIW, the tag name and the class name don't _have_ to line up in a normal Stencil project, but it might make sense here to minimize the cognitive overhead if they were different